src/pybreaker.py

```diff
@@ -44,8 +61,13 @@
 STATE_CLOSED = "closed"
 STATE_HALF_OPEN = "half-open"
 
+T = TypeVar("T")
+ExceptionType = TypeVar("ExceptionType", bound=BaseException)
+CBListenerType = TypeVar("CBListenerType", bound="CircuitBreakerListener")
```

Comment:
`bound` is used to say: "all subclasses are valid". Without this, it's not possible to type annotate fields that accept objects from subclasses.

Reply:
Makes sense, thanks for the explanation.